trajopt_sco/src/osqp_interface.cpp

```diff
@@ -102,7 +102,11 @@ void OSQPModel::updateObjective()
 
   Eigen::SparseMatrix<double> sm;
   exprToEigen(objective_, sm, q_, n, true);
-  eigenToCSC(sm, P_row_indices_, P_column_pointers_, P_csc_data_);
+
+  // Copy triangular upper into empty matrix
+  Eigen::SparseMatrix<double> triangular_sm;
+  triangular_sm = sm.triangularView<Eigen::Upper>();
+  eigenToCSC(triangular_sm, P_row_indices_, P_column_pointers_, P_csc_data_);
```

Comment:
Does this change the behavior. Before the whole matrix was passed and now only the upper triangle is?